doc/guide_3rd_party.md

```diff
@@ -176,10 +268,72 @@ Specify the amount to transfer to the receiver wallet. If needed the certificate
 
 By default hashed attributes are not included in the transfer. Make sure to include any hashed attributes, in the transfer request, the receiver wallet owner should be able to see.
 
+```csharp
+var token = "<access-token>"; // Access token obtained with client-credentials
+var organizationId = "<organizationId>"; // Organization id of sender
+
+var requestObject = new
+{
+    CertificateId = certificateFederatedStreamId, // the federatedStreamId from the get Certificates endpoint
+    ReceiverId = externalEndpointId, // The externalEndpointId created earlier
```

Comment:
The reason I changed it here, is it's a bug that ReceiverId. Because in fact it's not the receiver id and I found out when I tried to call with the receiver id :D

Reply:
Det virker forvirrende. Så hvorfor ikke fjerne receiverId helt?

Reply:
Ii can't remove ReceiverId because it's part of the API spec. I am just showing how to make the calls.